src/commands/email/send.js

```diff
@@ -0,0 +1,136 @@
+const { flags } = require('@oclif/command');
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const sgMail = require('@sendgrid/mail');
+
+class Send extends BaseCommand {
+  async run() {
+    await super.run();
+    if (!process.env.SENDGRID_API_KEY) {
+      this.logger.error('Make sure you have an environment variable called SENDGRID_API_KEY set up with your SendGrid API key. Visit https://app.sendgrid.com/settings/api_keys to get an API key.');
+      return this.exit(1);
+    }
+    await this.promptForFromEmail();
+    const validFromEmail = this.validateEmail(this.fromEmail);
+    const stringFromEmail = validFromEmail.toString();
```

Comment:
Why is `toString` needed here?

Reply:
`validateEmail()` returns an array of valid email addresses. The `from email address` needs to be a string in order to send correctly. I have now modified it to use the first item in the array for the sending address.